bin/run-bench.js

```diff
@@ -76,67 +84,74 @@ class JSONPrinter {
 
 run()
 
-function run() {
+const errorAndExit = (err, message, code) => {
```

Comment:
We have two very similar implementations of this function, one here and one in compare-bench-results.js, can we just create one?

Reply:
I was definitely leaning that way, but was biased toward not altering the expected behavior of the functions until I'd got them converted. Easy enough to do now!